src/keydra/providers/splunk_hec.py

```diff
@@ -62,8 +62,8 @@ def _rotate_secret(self, secret):
                 secret_id=secret['config']['rotatewith']['key']
             )
         )
-        username = operator_creds['key']
-        passwd = operator_creds['secret']
+        username = operator_creds['username']
+        passwd = operator_creds['password']
```

Comment:
Perhaps import the consts from Splunk provider to avoid such discrepancies in the future?

Reply:
Great idea - I didn't know you could do this! Updated.